newrelic/hooks/framework_pylons.py

```diff
@@ -16,14 +16,14 @@
 import newrelic.api.transaction_name
 import newrelic.api.function_trace
 import newrelic.api.error_trace
-import newrelic.api.object_wrapper
+import newrelic.common.object_wrapper
```

Comment:
All of these callable name usages should import from `newrelic.common.object_names`? At this point, callable_name isn't even imported in object_wrapper. The pylint linter caught this issue if you look at the megalinter logs. The tests didn't fail though cause we don't actually have tests for this file.

Reply:
I just pushed up a couple fixes for this.